### PR TITLE
Adding coming soon note

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8e38091](https://github.com/OfficeDev/office-scripts-docs/commits/8e380912910442671083023abcd1c19265b81133)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/preview-note.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-ComingSoon/docs/includes/preview-note.md) | :white_check_mark:Succeeded | |
[docs/index.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-ComingSoon/docs/index.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C16%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912161759590970-62%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=RR7%2FjBhjGCphQpbvEA38M9TRaeh82TnH4g%2BTI24Kr7M%3D&st=2019-12-16T17%3A56%3A09Z&se=2020-01-16T18%3A01%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Thanks @Reezaali. I've made the requested changes.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[04f0d55](https://github.com/OfficeDev/office-scripts-docs/commits/04f0d55f28b0cd3d0424eff08b748dfcef4ac4cf)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/preview-note.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-ComingSoon/docs/includes/preview-note.md) | :white_check_mark:Succeeded | |
[docs/index.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-ComingSoon/docs/index.md) | :white_check_mark:Succeeded | |
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-ComingSoon/docs/overview/excel.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-ComingSoon/docs/tutorials/excel-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C16%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912161832347447-62%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=iLntHSjzCW%2BZBA3ZvxfnOWA1sYlqk3M8MHuRTkipQm0%3D&st=2019-12-16T18%3A28%3A46Z&se=2020-01-16T18%3A33%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
This should also address #57

File: docs/tutorials/excel-tutorial.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This tutorial will teach you the basics of recording, editing, and writing an Of
 Before starting this tutorial, you'll need access to Office Scripts, which requires the following:
 
 - Excel on the web.
-- [Opt-in to the Office Scripts preview (admin-enabled)](https://aka.ms/office-scripts-admin). This adds the **Automate** tab to the ribbon.
+- Have your [admin enable the Office Scripts for your account](https://support.office.com/article/office-scripting-settings-in-m365-19d3c51a-6ca2-40ab-978d-60fa49554dcf). This adds the **Automate** tab to the ribbon.
```
Comment (Reezaali) [inline docs/tutorials/excel-tutorial.md]: Can we change the text to read:
Enable Office Scripts on your account, which adds the Automate tab to the ribbon
?

Comment (AlexJerabek) [reply docs/tutorials/excel-tutorial.md]: I cleaned that line up a bit, During the docs bash, it was asked we make it very clear the admin needs to activate this feature, not the end-user.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0e958b2](https://github.com/OfficeDev/office-scripts-docs/commits/0e958b20b0bb7cf87d9bf6205e1d99aed3a8c252)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-5952d72c)

<a id="5952d72c"></a>
###
 - **[Error]** `````Run script failed with no detailed information. Please retry building your content again. If the issue still happens, open a ticket in https://SiteHelp and include the error report for our team to troubleshoot`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C16%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912161841012500-62%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Off2xCPKJvW1dPI%2FLmS%2Fba83T%2FHwxSpPnqkY9Md9xFg%3D&st=2019-12-16T18%3A36%3A47Z&se=2020-01-16T18%3A41%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tutorials/excel-tutorial.md (hunk 1 above)
Comment (Reezaali) [reply docs/tutorials/excel-tutorial.md]: Got it. How about: 

Ask you administrator to enable Office Scripts on your account, which adds the Automate tab to the ribbon.

?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4acdac9](https://github.com/OfficeDev/office-scripts-docs/commits/4acdac93cdeefa7b833b5a1bc8dfad575443cb2a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/preview-note.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-ComingSoon/docs/includes/preview-note.md) | :white_check_mark:Succeeded | |
[docs/index.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-ComingSoon/docs/index.md) | :white_check_mark:Succeeded | |
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-ComingSoon/docs/overview/excel.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-ComingSoon/docs/tutorials/excel-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C16%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912161845317051-62%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=51e94zoyw65T2oASH5G78z5RhRz2NoGW2LDi1FLotN4%3D&st=2019-12-16T18%3A41%3A42Z&se=2020-01-16T18%3A46%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**